meson.build

```diff
@@ -57,6 +58,12 @@ if shellcheck.found()
   test('shellcheck toolbox (deprecated)', shellcheck, args: [toolbox_sh])
 endif
 
+if codespell.found()
+  test('codespell toolbox', codespell, args: [meson.current_source_dir() / 'src' , 
+  meson.current_source_dir() / 'cmd', meson.current_source_dir() / 'profile.d', meson.current_source_dir() / 'doc', 
+  meson.current_source_dir() / 'data'])
```

Comment:
Aren't we missing directories like `playbooks` and `test`?

Does codespell work recursively?  If not, then we are also missing directories like `data/config`.

Reply:
It seems like codespell does indeed work recursively, because:
```
$ git branch
...
* main
* ...
$ codespell .
./CONTRIBUTING.md:95: infrustructure ==> infrastructure
./builddir/doc/toolbox-rm.1:28: conjuction ==> conjunction
./builddir/doc/toolbox.1:77: TE ==> THE, BE, WE, TO
./builddir/doc/toolbox-rmi.1:28: conjuction ==> conjunction
./doc/toolbox-rmi.1.md:23: conjuction ==> conjunction
./doc/toolbox-rm.1.md:23: conjuction ==> conjunction
./test/system/105-enter.bats:46: specifc ==> specific
./test/system/libs/helpers.bash:73: throught ==> thought, through, throughout
./test/system/libs/bats-support/README.md:107: panicing ==> panicking
./src/cmd/completion.go:65: complet ==> complete
./src/cmd/initContainer.go:275: defult ==> default
```

So, I think we don't need to list out all the directories separately.  Only `meson.current_source_dir()` should be enough.

Reply:
The problem is that using meson.current_source_dir() generates spelling mistakes because of the 'builddir' directory. We have to find a way to exclude this directory and include the rest to avoid those spelling mistakes inside 'builddir'

Reply:
The `-S` and `-x` options could be helpful.

Reply:
@HarryMichal when I use the command ```codespell -S``` or ```codespell -x``` and I include the directory that I want to exclude from the test it doesn't work, it still shows the spelling mistakes inside that directory.

Reply:
Could you please update your branch with the exact changes you tried?

Also, use `codespell --skip ...`, not `codespell -S ...`.  The longer option is a lot easier to read than the shorter alias.